{"Functions": {"InternalError(java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#Throwable_E@", "Throwable: the cause (which is saved for later retrieval by the @B_java#lang#Throwable#getCause_E@ method). (A null value is permitted, and indicates that the cause is nonexistent or unknown.)"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an InternalError with the specified cause and a detail message of (cause==null ? null : cause.toString()) (which typically contains the class and detail message of cause).", "history": "Added in API level 24", "FullName": "public InternalError (Throwable cause)"}, "InternalError(java.lang.String,java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "String: the detail message (which is saved for later retrieval by the @B_java#lang#Throwable#getMessage_E@ method)."], ["@B_java#lang#Throwable_E@", "Throwable: the cause (which is saved for later retrieval by the @B_java#lang#Throwable#getCause_E@ method). (A null value is permitted, and indicates that the cause is nonexistent or unknown.)"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an InternalError with the specified detail message and cause. Note that the detail message associated with cause is", "history": "Added in API level 24", "FullName": "public InternalError (String message, Throwable cause)"}, "InternalError(java.lang.String)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "String: the detail message."]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an InternalError with the specified detail message.", "history": "Added in API level 1", "FullName": "public InternalError (String message)"}, "InternalError()": {"Returns": [], "Parameters": [], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an InternalError with no detail message.", "history": "Added in API level 1", "FullName": "public InternalError ()"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Error_E@", "@B_java#lang#VirtualMachineError_E@"], "ClassName": "java.lang.InternalError", "ClassDesc": "Thrown to indicate some unexpected internal error has occurred in the Java Virtual Machine."}